### ISSUE TITLE
"Find in Project" without a project crashes TM

### ISSUE THREAD

**Issue by triplef** [briancollins/nice_find#23]
When I select "Find in Project" when there is no project open in TextMate (e.g. no window, or just single-document windows), I get the following infinite loop which ultimately crashes TM. This is on 10.6.2 with nice_find 2.2.

```
1374 Thread_82451   DispatchQueue_1: com.apple.main-thread  (serial)
  261 -[NSWindowController showWindow:]
    261 -[NSWindow makeKeyAndOrderFront:]
      261 -[NSWindow _makeKeyRegardlessOfVisibility]
        261 -[NSWindow makeKeyWindow]
          261 -[NSWindow _changeKeyAndMainLimitedOK:]
            261 -[NSWindow becomeKeyWindow]
              261 -[NSNotificationCenter postNotificationName:object:]
                261 -[NSNotificationCenter postNotificationName:object:userInfo:]
                  261 _CFXNotificationPostNotification
                    261 __CFXNotificationPost
                      261 _nsnote_callback
                        261 -[FindController show]
                          261 -[NSWindowController showWindow:]
                            261 -[NSWindow makeKeyAndOrderFront:]
                              261 -[NSWindow _makeKeyRegardlessOfVisibility]
                                261 -[NSWindow makeKeyWindow]
                                  261 -[NSWindow _changeKeyAndMainLimitedOK:]
                                    261 -[NSWindow becomeKeyWindow]
                                      261 -[NSNotificationCenter postNotificationName:object:]
                                        261 -[NSNotificationCenter postNotificationName:object:userInfo:]
                                          261 _CFXNotificationPostNotification
                                            261 __CFXNotificationPost
                                              261 _nsnote_callback
                                                261 -[FindController show]
                                                  261 -[NSWindowController showWindow:]
                                                    ... (and so on)
```

**Comment by triplef** [comment]
Does anyone else have this problem? It’s always reproducible on my machine.

**Comment by briancollins** [comment]
dont crash when no project is open. closed by 7c38d920153e04cb0af160e39b09151c15a5eb6d.
